hmt_escrow/storage.py

```diff
@@ -1,31 +1,39 @@
+import os
 import logging
 import codecs
 import hashlib
 import json
-from typing import Dict, Tuple
-
 import ipfsapi
-import os
+import timeout_decorator
 
+from typing import Dict, Tuple
 from eth_keys import keys
 from p2p import ecies
+from ipfsapi import Client
 
 SHARED_MAC_DATA = os.getenv(
     "SHARED_MAC",
     b'9da0d3721774843193737244a0f3355191f66ff7321e83eae83f7f746eb34350')
 
 LOG = logging.getLogger("hmt_escrow.storage")
+IPFS_HOST = os.getenv("IPFS_HOST", "localhost")
+IPFS_PORT = int(os.getenv("IPFS_PORT", 5001))
 
-if not os.getenv("IPFS_DISABLE"):
-    _host = os.getenv("IPFS_HOSTNAME", 'localhost')
-    _port = int(os.getenv("IPFS_TCP_PORT", '5001'))
+
+@timeout_decorator.timeout(20)
+def _connect(host: str, port: int) -> Client:
     try:
-        API = ipfsapi.connect(_host, _port)
+        IPFS_CLIENT = ipfsapi.connect(host, port)
+        return IPFS_CLIENT
     except Exception as e:
         raise e
         LOG.error("Connection with IPFS failed because of: {}".format(e))
```

Comment:
Why would this logger ever fire?

Reply:
Should be fixed now, that what you meant?

Reply:
He's saying you'll never hit this line, might as well remove it or move it before the raise

Reply:
already moved it, wanted to make sure I understood it correctly

Reply:
Oh i see, I was looking at the filtered commit.  I don't see the raise anymore

Reply:
here: https://github.com/hCaptcha/hmt-escrow/blob/f2b88775cad5eaa5bfcca1e4011dc020949f1ec7/hmt_escrow/storage.py#L29

Reply:
lgtm